script/seed.js

```diff
@@ -12,14 +12,217 @@ async function seed() {
     password: 'firstPassword'
   })
 
-  const firstProduct = await Product.create({
-    price: 4.99,
-    imageUrl: 'someimage.com',
+  await Product.create({
+    name: 'Dunk-a-Roos',
+    price: 450,
+    imageUrl: 'https://i.redd.it/3phyzhloyxbz.jpg',
+    year: '1990',
+    description:
+      'Dunk-a-Roos are a snack food from Betty Crocker, first launched in 1990. It consists of a snack-sized package containing cookies and icing; the cookies are meant to be dunked into the icing before eating.',
+    manufacturer: 'Betty Crocker',
+    itemQuantity: 50
+  })
+  await Product.create({
+    name: 'Surge',
+    price: 275,
+    imageUrl:
+      'https://images-na.ssl-images-amazon.com/images/I/51Yd6SuGTIL.jpg',
+    year: '1996',
+    description:
+      'Surge is a citrus flavored soft drink first produced in the 1990s by The Coca-Cola Company to compete with Pepsi`s Mountain Dew.',
+    manufacturer: 'Coca Cola',
+    itemQuantity: 50
+  })
+  await Product.create({
+    name: 'Spice Girl Fantasty Ball Lollipop',
+    price: 190,
+    imageUrl:
+      'https://i.pinimg.com/originals/da/13/17/da131760d1fae4bc7ef444a8173227de.jpg',
+    year: '1997',
+    description:
+      'Spice Girls Official Product! One lollipop of each member ~ Posh, Baby, Scary, Sporty and Ginger Spice in original packing. Each one includes a sticker.',
+    manufacturer: 'Chupa Chupa',
```

Comment:
Is Chupa Chupa supposed to be Chupa Chups? I think Chupa Chupa is supposed to be Chupa Chups.